### PR TITLE
[prometheus-kafka-exporter] Annotate tests/* with "helm.sh/hook: test" so that we don't launch stray Kafka/ZooKeeper pods.

### PR DESCRIPTION
**Pull request by vitaliyf**
#### What this PR does / why we need it:

Currently, deploying this chart leaves behind pods running Kafka and ZooKeeper.

#### Which issue this PR fixes

n/a

#### Special notes for your reviewer:

This change was mentioned by @scottrigby when this chart was contributed here: https://github.com/prometheus-community/helm-charts/pull/91#issuecomment-695144728 but never added.

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by gkarthiks** [comment]
why not have https://github.com/prometheus-community/helm-charts/pull/289 as well in the same chart version?

**Comment by vitaliyf** [comment]
> why not have #289 as well in the same chart version?

Done. Mostly because I created this PR as I was settings things up and the other after I was done and thought some things I discovered would be helpful to others.

File: charts/prometheus-kafka-exporter/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: "v1.2.0"
 description: A Helm chart to export the metrics from Kafka in Prometheus format using the kafka-exporter from https://github.com/danielqsj/kafka_exporter
 name: prometheus-kafka-exporter
 home: https://github.com/danielqsj/kafka_exporter
-version: 0.1.4
+version: 0.1.5
```
Comment (gkarthiks) [inline charts/prometheus-kafka-exporter/Chart.yaml:6]: Hi, why don't you rebase and raise the chart verison?

**Review by monotek — CHANGES_REQUESTED** [review]
@vitaliyf  please check the failing chart install test

**Pull request closed without merging**